README.md

```diff
@@ -732,6 +733,12 @@ Software written in Go.
 * [webhook](https://github.com/adnanh/webhook) - Tool which allows user to create HTTP endpoints (hooks) that execute commands on the server
 * [Wide](https://wide.b3log.org) - A Web-based IDE for Teams using Golang.
 
+## Resource Embedding
+
+* [go.rice](https://github.com/GeertJohan/go.rice) - go.rice is a Go package that makes working with resources such as html,js,css,images and templates very easy. 
+* [bindata](https://github.com/jteeuwen/go-bindata) - Package that converts any file into managable Go source code. 
+* [go-resources](https://github.com/omeid/go-resources) - Unfancy resources embedding with Go.
+
```

Comment:
This section is still not alphabetical. It should come after Package Management section and before Science and Data Analysis section.